sdk/textanalytics/ai-text-analytics/samples/v5/javascript/beginAnalyzeBatchActions.js

```diff
@@ -36,20 +36,21 @@ async function main() {
   const poller = await client.beginAnalyzeBatchActions(documents, actions, "en", {
     includeStatistics: true
   });
+  const resultPages = await poller.pollUntilDone();
```

Comment:
putting the await up here seems like it would prevent the progress callback from firing ever?

Reply:
CC @deyaaeldeen

Reply:
@witemple-msft I agree with @xirzec on this, is there a reason to push it to here? it looks like you were working off an older history and merging latest changes was not correct.

Reply:
I think you need to make sure there are no changes to the samples other than the addition of the `@summary` tag.

Reply:
Seems like I might've done the migration, and then while the PR has been in flight, some updates were made to the TA samples.

Reply:
Actually this is how the sample is in the TypeScript samples currently in master. Since JS samples are now generated from the TS samples, the JS sample was changed to mirror the TS sample.  I'll fix this so that the TS source is the way the current JS master is.

Reply:
👍 

with your work, we no longer have to worry about inconsistencies between both versions